app/controllers/user_badges_controller.rb

```diff
@@ -1,13 +1,15 @@
 # frozen_string_literal: true
 
 class UserBadgesController < ApplicationController
+  MAX_FAVORITES = 2
+
   before_action :ensure_badges_enabled
 
   def index
     params.permit [:granted_before, :offset, :username]
 
     badge = fetch_badge_from_params
-    user_badges = badge.user_badges.order('granted_at DESC, id DESC').limit(96)
+    user_badges = badge.user_badges.order("granted_at DESC, id DESC").limit(96)
```

Comment:
Any chances you could extract the limit into a constant as well?

Reply:
Done!